action/kafkaFeed.js

```diff
@@ -17,6 +17,11 @@ function main(params) {
     var namespace = encodeURIComponent(triggerComponents[1]);
     var trigger = encodeURIComponent(triggerComponents[2]);
 
+    if (namespace === '_') {
+        whisk.error('You must supply a non-default namespace.');
```

Comment:
Cannot use the environment variable as the user name be using a namespace with "_dev" in it, but process.env['__OW_NAMESPACE'] will not return a namespace with "_dev".